.github/workflows/new-cfe-cherry-pick.yml

```
@@ -238,15 +238,19 @@ jobs:
             - name: Comment on PR about the failed cherry pick
               if: ${{ failure() }}
               run: |
-                  gh pr comment $HTML_URL --body "Cherry picking failed for the trunk. Please do it manually. Or debug, what happened and run the workflow again. Workflow link for debugging: https://github.com/${{ github.repository_owner }}/${{ github.repository }}/actions/workflows/new-cfe-cherry-pick.yml"
+                  gh pr comment $HTML_URL --body "IMPORTANT: Merging this PR to appropriate branchs is critical to the release process and ensures that the bug does not regress in the future releases.
```

```suggestion
                  gh pr comment $HTML_URL --body "IMPORTANT: Merging this PR to appropriate branches is critical to the release process and ensures that the bug does not cause regressions in the future releases.
```

Minor/non-essential.